src/content/blog/2024_mitiq_impact.md

```diff
@@ -0,0 +1,147 @@
+> <p style="font-size: 24px;">Error mitigation will continue to be useful in the Megaquop era and beyond.</p>
+
+Preskill’s remarks underscore the enduring importance of error mitigation and highlight the need for an open-source, platform-agnostic tool like Mitiq to support the growing quantum ecosystem.
+When Mitiq was first introduced in 2020, the landscape of QEM tools was nascent, with early efforts like pyquil’s measurement error mitigation and Qiskit Ignis paving the way.
+Since then, the ecosystem has expanded, with Mitiq leading the charge by implementing nine QEM techniques and offering official support for 6 quantum SDKs (and un-official support for more).
+
+Let’s take a closer look at how Mitiq has shaped the field and grown over time.
+
+
+## Adoption and Community Growth
+
+At the core of making QEM available to all, is building a platform-agnostic tool.
+Whether you're writing your quantum program with Qiskit, Cirq, PyQuil, Qibo, Braket, ..., you can use Mitiq!
+This has been a major driving force behind the growth and adoption of the project, and allows for users to write QEM code once with the flexibility of running on any (gate-based) hardware.
+As the only SDK-agnostic QEM tool[^2], Mitiq’s adoption metrics provide a clear measure of its utility and impact.
+
+[^2]: To create such a tool in the ever-changing quantum software landscape that does not have an established IR, is, to say the least, a lot of work. In particular the **maintenance** overhead required to stay up to date with $n$ SDKs is quadratic.
+
+- **Downloads:** Over 195,000 downloads (and counting!).
+- **GitHub Engagement:** Mitiq has accumulated over 350 stars, 160 forks, and contributions from 76 developers worldwide.
+- **Scientific Reach:** Researchers have cited Mitiq over 120 times since 2020 when the [whitepaper](https://arxiv.org/abs/2009.04417) was put online.
+
+<div style="display: flex; justify-content: center; align-items: center;" class="side-by-side">
+  <figure style="text-align: center;">
+    <img src="/images/2024-mitiq/mitiq.png" width="600" alt="Bar chart of the Mitiq downloads per year from 2020 to 2024.">
+    <figcaption>More than 50% growth in Mitiq downloads year over year!</figcaption>
+  </figure>
+</div>
+
+The thriving Mitiq community is a testament to the importance of open-source collaboration.
+From first-time contributors to seasoned quantum researchers, everyone has played a role in making Mitiq a vital tool for error mitigation.
+
+## Driving Research and Innovation
+
+Mitiq has been cited in over 120 peer-reviewed research papers, enabling progress in areas such as condensed matter, quantum simulation, and benchmarking.
+Some notable examples from the past 4 years include:
+
+- [**Computational physics:**](https://arxiv.org/abs/2112.12654) Researchers used Mitiq's ZNE functionality to mitigate noise in quantum simulations of Heisenberg spin chains, enhancing fidelity and extending simulations beyond hardware coherence times.
```

Comment:
As a reader of the rendered version, it's not clear to me that those are links leading to papers. Maybe you want to add the actual title of the paper as text of the link.

Reply:
Good point. I like your suggestion.